oauth2_http/javatests/com/google/auth/oauth2/ImpersonatedCredentialsTest.java

```diff
@@ -690,7 +908,7 @@ public void serialize() throws IOException, ClassNotFoundException {
     assertSame(deserializedCredentials.clock, Clock.SYSTEM);
   }
 
-  private String getDefaultExpireTime() {
+  public static String getDefaultExpireTime() {
     Date currentDate = new Date();
```

Comment:
why set the date? Calendar.getInstance is already the current time.

Reply:
Done